doc/devdocs/core/settings/compatibility-legacy-settings.md

```diff
@@ -9,4 +9,4 @@ The following must be kept in mind regarding compatibility with settings v1 and
 - The status of each of the modules is communicated with the runner in the form of a json object. The names of all the powerToys is set in the [`EnableModules.cs`](src/settings-ui/Settings.UI.Library/EnabledModules.cs) file. The `JsonPropertyName` must not be changed to ensure that the information is dispatched properly to all the modules by the runner.
 
 ### ImageResizer anomaly
-All the powertoys have the same folder name as well as JsonPropertyName to communicate information with the runner. However that is not the case with ImageResizer. The folder name is `ImageResizer` whereas the JsonPropertyName is `Image Resizer`(Note the additional space). This should not be changed to ensure backward compatibility as well as proper functioning of the module.
+All the powertoys have the same folder name as well as JsonPropertyName to communicate information with the runner. However that is not the case with ImageResizer. The folder name is `ImageResizer` whereas the JsonPropertyName has an additional space: `Image Resizer`. This should not be changed to ensure backward compatibility as well as proper functioning of the module.
```

Comment:
A rather new, but not particularly rule complained about this -- it doesn't like sentence like things in parentheticals that don't also include periods/other final punctuation marks.

I think this rewording improves things.